connectors/sources/tests/test_confluence.py

```diff
@@ -298,6 +315,38 @@ async def test_close_without_client_session():
     assert source.confluence_client.session is None
 
 
+@pytest.mark.asyncio
+async def test_verify_spaces_when_space_keys_are_valid():
+    """Test verify_spaces method with valid space keys"""
```

Comment:
nit: 
```suggestion
```

IMO we should remove those comments, if they're basically having the same information as the test function name